.github/workflows/ci-core.yml

```diff
@@ -88,9 +88,15 @@ jobs:
           only-new-issues: ${{ github.event.schedule == '' }} # show only new issues, unless it's a scheduled run
           allow-extra-out-format-args: true
           args: --out-format checkstyle:golangci-lint-report.xml
+      - name: Install bat
+        uses: awalsh128/cache-apt-pkgs-action@1850ee53f6e706525805321a3f2f863dcf73c962 # v1.3.0
```

Comment:
How much of an improvement does this give us over just reinstalling the package everytime? I'd like to keep our GHA deps low if we can.

Another route would be to install a dockerized version since I'd imagine this image already has docker

Reply:
I haven't actually gotten it to work yet. IIRC I was just trying different approach of calling `bat`, and had no luck. Feel free to push a new solution.

> bat: command not found

Reply:
If the stdout format is already colour coded, i'd specify it as an additional output instead. https://github.com/golangci/golangci-lint/pull/2386